cmake/server.cmake

```diff
@@ -0,0 +1,51 @@
+if(WIN32)
+  set(BASEPYEXE py.exe -3)
+  set(SERVER_DIR ${CMAKE_CURRENT_BINARY_DIR}/veles-server)
+  set(SERVER_PYTHON_DIR ${SERVER_DIR}/python)
+  file(TO_NATIVE_PATH ${SERVER_PYTHON_DIR} SERVER_PYTHON_DIR_NATIVE)
+
+  if(NOT EMBED_PYTHON_ARCHIVE_PATH)
+    set(EMBED_PYTHON_URL "https://www.python.org/ftp/python/3.6.1/python-3.6.1-embed-win32.zip")
```

Comment:
Hmm, I've noticed there is also `python-3.6.1-embed-amd64.zip` - since we have 2 Windows distributions (32 and 64bits) can we maybe use 64bit version for 64bit distribution?